app_controller/pixi_frontend/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "outDir": "./src/js/src",
         "allowJs": true,
-        "target": "es5"
+        "target": "es5",
```

Comment:
Why is our target es5?

Reply:
Ah, without babelify, browserify only seems to work with es5. I just left it because I wanted to work on the code first, let me fix that.

Reply:
gotcha. no it's fine to target that with it. I was just curious